cosmos/models/Stage.py

```diff
@@ -14,7 +14,9 @@
 
 @signal_stage_status_change.connect
 def stage_status_changed(stage):
-    stage.log.info('%s %s (%s/%s Tasks were successful)' % (stage, stage.status, sum(t.successful for t in stage.tasks), len(stage.tasks)))
```

Comment:
Thanks, I never would have thought of that! I didn't create a ve, I just relied on the travis builds which are presumably building from scratch.